specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json

```diff
@@ -208,7 +208,7 @@
         },
         "translator": {
           "description": "Copy activity translator. If not specified, tabular translator is used.",
-          "$ref": "#/definitions/CopyTranslator"
```

Comment:
It's not `CopyTranslator` anymore?

Reply:
This is again to support Expressions. @Yiliu-microsoft  please make sure the description of the property has the accepted type. Look at other properties that are expressions.